samples/net/lwm2m_client/README.rst

```diff
@@ -106,6 +109,32 @@ Setup DTLS security in Leshan Demo Server:
     Key: 000102030405060708090a0b0c0d0e0f
 - Start the Zephyr sample
 
+Bluetooth Support
+=================
+
+To build the lwm2m-client sample for hardware requiring Bluetooth for
+networking (IPSP node connected via 6lowpan) do the following:
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/net/lwm2m_client
+   :host-os: unix
+   :board: <board to use>
+   :conf: prj.conf overlay-bt.conf
+   :goals: build
+   :compact:
+
+The overlay-*.conf files can also be combined.  For example, you could build a
```

Comment:
ReST thinks the ``*`` is starting an italics section and isn't happy about not seeing an end.  Fix this by escaping:

    The overlay-\*.conf files ...